flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.05";
+    utils.url = "github:gytis-ivaskevicius/flake-utils-plus";
+  };
+
+  outputs = inputs@{ self, utils, ... }:
+    utils.lib.mkFlake rec {
+      inherit self inputs;
+
+      supportedSystems = [
+        "aarch64-linux"
+        "aarch64-darwin"
+        "i686-linux"
+        "x86_64-darwin"
+        "x86_64-linux"
+      ];
+
+      outputsBuilder = channels: with channels; {
+        packages = with nixpkgs; { 
+          inherit (nixpkgs) package-from-overlays;
+
+          nixpacks = rustPlatform.buildRustPackage rec {
+            pname = "nixpacks";
+            version = "v0.0.20";
```

Comment:
Could the release action potentially automatically bump this version here? I'm not sure if it is feasible to do, but that would make it much more helpful rather than worrying about updating the version in this file every time a new release happens.

Reply:
actually `nixpkgs` actions do exactly this although i never searched how. I'll try to reproduce

Reply:
So, for `nixpkgs` repo they use this [package](https://github.com/r-ryantm/nixpkgs-update) and a bot user.

We can build soem similar and integrate with CI, that's make sense?

Reply:
Yup that works! Acceptance criteria are that when a new version of nixpacks has released the version in the flake is automatically updated.

Reply:
I implement a new CI action to bump the version on a new tag release!